pkg/common/model.go

```diff
@@ -15,16 +15,16 @@ type RepoInfo struct {
 }
 
 type RepoSummary struct {
-	Name          string       `json:"name"`
-	LastUpdated   time.Time    `json:"lastUpdated"`
-	Size          string       `json:"size"`
-	Platforms     []Platform   `json:"platforms"`
-	Vendors       []string     `json:"vendors"`
-	IsStarred     bool         `json:"isStarred"`
-	IsBookmarked  bool         `json:"isBookmarked"`
-	StarCount     int          `json:"starCount"`
-	DownloadCount int          `json:"downloadCount"`
-	NewestImage   ImageSummary `json:"newestImage"`
+	Name          string       `json:"Name"`          //nolint:tagliatelle // graphQL schema
```

Comment:
Why was this change needed?

Reply:
Previously, the GQL server was handling serialization of the data for responding to the client. Now, since the proxy handler is responding on behalf of the GQL server (after proxying), the standard Golang serialization takes place.

GQL starts keys with an uppercase, but the standard struct annotations don't encourage the same - hence the `nolint`.

Example:
```
{
    "errors": [
        {
            "message": "unable to run vulnerability scan on tag v0.0.19.231225-squashfs in repo machine/bootkit/rootfs: error: image 'machine/bootkit/rootfs@sha256:9efb1b9dd349e3fc5fa2dd658354a395a28ac7e391c85a72b39384da7a5ec7a1' scanning is not supported for given image media type",
            "path": [
                "GlobalSearch"
            ]
        },
        {
            "message": "unable to run vulnerability scan in repo machine/bootkit/rootfs: manifest digest: sha256:9efb1b9dd349e3fc5fa2dd658354a395a28ac7e391c85a72b39384da7a5ec7a1, error: image 'machine/bootkit/rootfs@sha256:9efb1b9dd349e3fc5fa2dd658354a395a28ac7e391c85a72b39384da7a5ec7a1' scanning is not supported for given image media type",
            "path": [
                "GlobalSearch"
            ]
        }
    ],
    "data": {
        "GlobalSearch": {
            "Page": {
                "TotalCount": 37,
                "ItemCount": 3
            },
            "Repos": [
                {
                    "Name": "machine/bootkit/rootfs",
                    "LastUpdated": "2023-12-25T15:31:59.110429376Z",
                    "Size": "319472431",
                    "Platforms": [
                        {
                            "Os": "linux",
                            "Arch": "amd64"
                        }
                    ],
                    "IsStarred": false,
                    "IsBookmarked": false,
                    "NewestImage": {
                        "Tag": "v0.0.19.231225-squashfs",
                        "Vulnerabilities": {
                            "MaxSeverity": "",
                            "Count": 0
                        },
                        "Description": "A minimal bootable root filesystem",
                        "IsSigned": false,
                        "SignatureInfo": [],
                        "Licenses": "GPLv2 and others",
                        "Vendor": "project-machine",
                        "Labels": ""
                    },
                    "StarCount": 0,
                    "DownloadCount": 5285
                },
                {
                    "Name": "c3/ubuntu/base-amd64",
                    "LastUpdated": "2024-03-01T00:46:16.186838886Z",
                    "Size": "273201849",
                    "Platforms": [
                        {
                            "Os": "linux",
                            "Arch": "amd64"
                        }
                    ],
                    "IsStarred": false,
                    "IsBookmarked": false,
                    "NewestImage": {
                        "Tag": "jammy",
                        "Vulnerabilities": {
                            "MaxSeverity": "",
                            "Count": 0
                        },
                        "Description": "base is a minimal glibc-based Linux system",
                        "IsSigned": false,
                        "SignatureInfo": [],
                        "Licenses": "GPL-2.0-or-later",
                        "Vendor": "Cisco Systems, Inc.",
                        "Labels": ""
                    },
                    "StarCount": 0,
                    "DownloadCount": 260
                },
                {
                    "Name": "tools/busybox",
                    "LastUpdated": "2022-10-04T18:22:45.289257759Z",
                    "Size": "773920",
                    "Platforms": [
                        {
                            "Os": "linux",
                            "Arch": "amd64"
                        }
                    ],
                    "IsStarred": false,
                    "IsBookmarked": false,
                    "NewestImage": {
                        "Tag": "1.34.1",
                        "Vulnerabilities": {
                            "MaxSeverity": "",
                            "Count": 0
                        },
                        "Description": "",
                        "IsSigned": false,
                        "SignatureInfo": [],
                        "Licenses": "",
                        "Vendor": "",
                        "Labels": ""
                    },
                    "StarCount": 0,
                    "DownloadCount": 112
                }
            ]
        }
    }
}
```